### PR TITLE
Feature/tda 2

### PR DESCRIPTION
**Pull request by duprijil**

### CODE REVIEW

File: tests/test_ted_sws_library.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+TDA_REPOSITORY_NAME = "tda_repository"
+
+
+def test_ted_sws_library():
+    mongodb_client = MongoClient(config.MONGO_DB_AUTH_URL)
+    notice_repository = NoticeRepository(mongodb_client=mongodb_client)
+    notice_list = list(notice_repository.list())
+    notices_distilled_rdf_manifestation = {notice.ted_id: notice.distilled_rdf_manifestation.object_data
+                                           for notice in notice_list if notice.distilled_rdf_manifestation}
+    print('')
```
Comment (CaptainOfHacks) [inline tests/test_ted_sws_library.py:15]: Don't use **print** in tests, in tests we use **assert** for check something.

File: tests/test_ted_sws_library.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    agraph_repositories = agraph_ts.list_repositories()
+
+    if TDA_REPOSITORY_NAME not in agraph_repositories:
+        agraph_ts.create_repository(repository_name=TDA_REPOSITORY_NAME)
+
+    print("Start load rdf content")
+    for notice_id, rdf_content in notices_distilled_rdf_manifestation.items():
+        print(f"Start load rdf content for notice_id={notice_id}")
+        agraph_ts.add_data_to_repository(file_content=rdf_content, mime_type="ttl")
+        print(f"Finish load rdf content for notice_id={notice_id}")
```
Comment (CaptainOfHacks) [inline tests/test_ted_sws_library.py:34]: This test need to check if same data arrive from Mongo to Agraph.

**Review by CaptainOfHacks — CHANGES_REQUESTED** [review]
Good, revise comments.

File: notebooks/queries/q1.rq [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  	?Contract epo:bindsBuyer ?Buyer.
+  			?Buyer rdf:type epo:Buyer;
+                   epo:playedBy ?Organization.
+  			?Organization rdf:type org:Organization;
+                          legal:registeredAddress ?Address.
+  			?Address rdf:type locn:Address;
+                     epo:hasCountryCode ?BuyerCountry.
+  
+
+} group by ?BuyerCountry
```
Comment (CaptainOfHacks) [inline notebooks/queries/q1.rq:32]: I recommend to create a resource folder at level with notebooks, and put queries in resources/sparql_queries/*.rq .

**Review by CaptainOfHacks — APPROVED** [review]
Good, revise comments before merge.

File: notebooks/queries/q1.rq (hunk 3 above)
Comment (duprijil) [reply notebooks/queries/q1.rq]: the resource folder was added and queries was moved there

**Pull request merged**